contents/docs/integrate/server/go.mdx

```diff
@@ -191,8 +191,13 @@ client.Enqueue(posthog.Capture{
 To check if a feature flag is on for a given user, you can call `isFeatureEnabled`, passing the flag's key and the user's distinct ID. You can optionally pass a third argument to override the default result to be returned if the flag is not found. This is set to `false` by default.
```

Comment:
consider replacing this entire section from ruby / python / node, and replacing the code pieces right after, since the above is out of date (specifically, defaults here don't exist anymore)